### ISSUE TITLE
Rendering issues with vim + tmux

### ISSUE THREAD

**Issue by eriols** [jwilm/alacritty#2439]
Which operating system does the issue occur on? MacOS 10.14.4

If on linux, are you using X11 or Wayland? No

Using tmux 2.9 and vim. After a while Alacritty 0.3.2 starts just painting everything black below the cursor. Eg if you vim gg to the top the whole screen goes black. It flickers a bit so sometimes you can move a few lines without issues but then it comes back. Once we have gotten into this state it continues for the duration of the session.
Same settings work without issues in eg iTerm.

Added two screenshots, one where the screen is black underneath the cursor and one where it looks normal, for comparison.

![broken_alacritty](https://user-images.githubusercontent.com/3170971/57704739-cd9cf900-7662-11e9-9b53-aa71e9110223.png)
![working_alacritty](https://user-images.githubusercontent.com/3170971/57704755-d4c40700-7662-11e9-9216-f99165b26495.png)

**Comment by chrisduerr** [comment]
Can you reproduce this with the latest git master?

**Comment by eriols** [comment]
@chrisduerr built it now, will let you know tomorrow! thanks.

**Comment by lovesegfault** [comment]
I tested on my Mac and was able to reproduce this issue (on the latest release.) Testing master now.

**Comment by lovesegfault** [comment]
I was unable to reproduce this consistently on the latest release, and in my ~20min of testing I was unable to reproduce this once on master. May or may not be fixed. Heisenbug.

**Comment by kchibisov** [comment]
I think it's just duplicate of #2353 , the screenshots and describe behaviors are quite similar. Also this can be checked by putting some debug output [here](https://github.com/jwilm/alacritty/blob/4fbae5e397c514c245432a6415a4c99c52b7978f/src/renderer/mod.rs#L1056) ( on v0.3.2), your rendering should become broken after you see your debug message.

**Comment by eriols** [comment]
Thanks for your help. I have run this on the latest git master all day (7 hours or so) without any issue so looks good. Feel free to close if you see no reason not to.

**Comment by aseng-avant** [comment]
I have this issue too, if this is fixed in master, do you know when a new release will come out?

**Comment by chrisduerr** [comment]
This is indeed a duplicate of #2353 which is why I asked if the latest master fixes it.

A new release is currently blocked by some upstream changes in the winit/glutin dependency.
